azurerm/internal/services/redisenterprise/redis_enterprise_cluster_resource.go

```diff
@@ -161,11 +157,11 @@ func resourceRedisEnterpriseClusterCreate(d *schema.ResourceData, meta interface
 		parameters.Zones = azure.ExpandZones(v.([]interface{}))
 	}
 
-	// RP currently does not return this value but will in the near future
-	// https://github.com/Azure/azure-sdk-for-go/issues/14420
-	// if v, ok := d.GetOk("minimum_tls_version"); ok {
-	// 	parameters.ClusterProperties.MinimumTLSVersion = redisenterprise.TLSVersion(v.(string))
-	// }
+	if v, ok := d.GetOk("minimum_tls_version"); ok {
```

Comment:
Since this is an root-level optional property with default value, which means the `d.GetOk()` will always return a value for it. Rather than doing a post check and set here, can we simply inline this property when constructing the `parameters` instead?